docs/src/charmil_i18n.md

```diff
@@ -7,30 +7,57 @@ Localization has never been easier. Charmil provides out of the box support for
 
 ## How to use
 
-1. Provide the default language, the path to your locales, and the file format of your locales to initialize the localizer. Charmil accepts locales in the yaml, toml, and json formats.
+1. Initialize a variable using go:embed, for storing the locales directory
```

Comment:
Could you add some more explanation on what this does and how embed works. Nothing too long as it is a stdlib feature but just a small note on what is going on here.

```suggestion
1. Embed provides access to files embedded in the running Go program meaning you can get your locales directory with it's the path tp it.  
```